test/e2e/resources/ecr_repository.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: ecr.services.k8s.aws/v1alpha1
```

Comment:
@A-Hilaly mind renaming this file to just `repository.yaml` please? The `ecr_` prefix is redundant :)

Reply:
++ fixed